test/integration.html

```diff
@@ -12,8 +12,20 @@
 <head>
   <title>iron-location</title>
 
-  <script src="../../webcomponentsjs/webcomponents.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <script src="../../web-component-tester/browser.js"></script>
+  <script>
```

Comment:
edge is the only browser that does not convert spaces into `%20` in the query params. the polyfill does not check for this.